README.md

```diff
@@ -180,6 +181,22 @@ Override schema:
     tag: string      # The image tag to use
 ```
 
+#### Overriding Remote Tag using CLI
+
+Additionally, you can run a docker image with a specific remote tag using the CLI. An example of doing so looks like this:
+
+```
+sso tb up venue-config-service -t venue-config-service:my_branch
```

Comment:
`my_branch` => `my_tag`

Reply:
Can we find a way to not need to duplicate the service name, if you are only running a single service? ie

`sso tb up venue-config-service -t my_tag`

`sso tb up -p vcs-deps -t postgres:m_tag1,redis:my_tag2`

Reply:
got it working btw